versionist.conf.js

```diff
@@ -1,4 +1,4 @@
-var _ = require('lodash');
+var _ = require(`${__dirname}/node_modules/lodash`);
```

Comment:
Perhaps `${__dirname}/node_modules` could be stored in a 'global variable' that versionist itself supplies to `versionist.conf.js` ?